libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -156,7 +237,12 @@ class AP_Motors {
     virtual void        set_update_rate( uint16_t speed_hz ) { _speed_hz = speed_hz; }
 
     // init
-    virtual void        init(motor_frame_class frame_class, motor_frame_type frame_type) = 0;
+    void                initialise(motor_frame_class frame_class, motor_frame_type frame_type)
```

Comment:
Ah.  We should probably change the name in this class rather than the externally visibly one.  "_init" is sometimes used :-)  Or "init_subclass"

Reply:
I did it this way to avoid having to change every subclass. There are only 2 or 3 external usages of the old init() vs. 6 subclasses and 12 source files involved in overriding it.
But I see now that the init method is public in all the subclassses, so making it private would mean touching all of those files anyway. Guess I'll go back to renaming init to a private _init and edit all of the subclasses :(